vars/RedHat.yml

```diff
@@ -2,7 +2,7 @@
 # File: vars/RedHat.yml - Red Hat OS variables for Nomad
 
 nomad_os_packages:
-  - "{% if (ansible_distribution == 'AlmaLinux' and ansible_distribution_version is version('9', '>=')) %}curl-minimal{% else %}curl{% endif %}"
+  - "{% if (ansible_distribution == 'AlmaLinux' and ansible_distribution_version is version('9', '>=')) or (minimal_curl is defined) %}curl-minimal{% else %}curl{% endif %}"
```

Comment:
We should not use two different ways to tackle this problem. Either use the variable `minimal_curl` or rely on the distribution facts. Both seem okay, but I'd probably use the distribution-vars since then we have no additional variable.

Reply:
That'd be great, but the problem is some AlmaLinux or Rocky setups use curl, while some use curl-minimal. The distribution setup isn't consistent.

Reply:
We could solve this by changing the task `Include OS variables" to something like this:

```
- name: Fetch OS dependent variables
  ansible.builtin.include_vars:
    file: "{{ item }}"
  with_first_found:
    - files:
        - "{{ ansible_facts.distribution }}_{{ ansible_facts.distribution_major_version }}.yml"
        - "{{ ansible_facts.distribution }}.yml"
        - "{{ ansible_facts.os_family }}_{{ ansible_facts.distribution_major_version }}.yml"
        - "{{ ansible_facts.os_family }}.yml"
      skip: true
  tags: always
```

Then we could seperate different os-versions. I guess even without any breaking changes. Do you want to take a stab at this?

Reply:
I've got a middle ground I'm trying.

Reply:
@rndmh3ro Maybe something like this? Where the default is in defaults, and we leave it for Alma 9 for backwards compatibility, but otherwise you can just override it.